core/config/locales/en.yml

```
@@ -1084,6 +1090,11 @@ en:
     generate_code: Generate coupon code
     get_back_to_the: Get back to the
     gift_card: Gift card
+    gift_card_batch: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)